src/Tinkoff.ISA.Providers.Jira.Tests/JiraProviderTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Atlassian.Jira;
+using Atlassian.Jira.Remote;
+using Microsoft.Extensions.Options;
+using Moq;
+using Tinkoff.ISA.Core;
+using Xunit;
+using JiraClient = Atlassian.Jira.Jira;
+
+namespace Tinkoff.ISA.Providers.Jira.Tests
+{
+    public class JiraProviderTests
+    {
+        private readonly IKnowledgeProvider _jiraProvider;
+        private readonly Mock<IIssueService> _issueServiceMock = new Mock<IIssueService>();
```

Comment:
why not in constructor?

Reply:
In order to write less :) Does it look ugly?

Reply:
It is not ugly but in all tests we require the same object and there is no need to reset the object?

Reply:
I reckon there is no need to recreate the mock object every time. We could just use `Setup(...)` for some method if needed.